File: src/statesman/commands/GotoBaseCommand.java
package statesman.commands;

public class GotoBaseCommand extends Command {

    public static final String ID = "gotob";

    public GotoBaseCommand() {
    }

    @Override
    public Command createInstance(String[] arguments) {
        return new GotoBaseCommand();
    }

    @Override
    public void execute() {
    }

}

File: src/Commands/GotoBaseCommand.cs
namespace Statesman.Commands
{
    public class GotoBaseCommand : Command
    {
        public static readonly string ID = "gotob";

        public GotoBaseCommand()
        {
        }

        public override Command CreateInstance(string[] arguments)
        {
            return new GotoBaseCommand();
        }
    }
}
